<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TAlarm</class>
 <widget class="QWidget" name="TAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="10" column="0">
       <widget class="QCheckBox" name="chbComPrm">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLedIndicator" name="ledModel61850" native="true"/>
      </item>
      <item row="4" column="1">
       <widget class="QLedIndicator" name="ledFaultChannel" native="true"/>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="chbWarning">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLedIndicator" name="ledTest61850" native="true"/>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="chbComPrd">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Неисправность канала</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QColorButton" name="pbPrmOut">
        <property name="text">
         <string>ПРМ выведен на сигнал</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLedIndicator" name="ledWarning" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chbFaultChannel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="chbFault">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QColorButton" name="pbComPrm">
        <property name="text">
         <string>Срабатывание ПРМ</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QColorButton" name="pbWarning">
        <property name="text">
         <string>Предупреждение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Моделирование 61850</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLedIndicator" name="ledFault" native="true"/>
      </item>
      <item row="13" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2">
       <widget class="QColorButton" name="pbFault">
        <property name="text">
         <string>Неисправность устройства</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Тест 61850</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QColorButton" name="pbComPrd">
        <property name="text">
         <string>Срабатывание ПРД</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>БВП</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>БСП</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>КВП</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLedIndicator</class>
   <extends>QWidget</extends>
   <header location="global">widget/qledindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>widget/qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
